scripts/unit-test/docker-compose.yml

```diff
@@ -7,10 +7,10 @@ services:
       context: ../../
       dockerfile: scripts/Dockerfile
       args:
-        - LOGSTASH_VERSION=${LOGSTASH_VERSION:-7.13.2}
```

Comment:
why are we changing this default value to 8.x?

Reply:
I made the change because its the latest version of Logstash and it might be good to test features/fixes against it given user interest in these versions as seen in issues like #149.  Let me know if its better to keep the old version.

Reply:
No strong opinion against this change, just that we might end up in updating this for every new version.